aiogoogle/auth/managers.py

```diff
@@ -92,6 +92,11 @@
 except ValueError:  # pragma: NO COVER
     GCE_METADATA_DEFAULT_TIMEOUT = 3
 
+try:
+    HTTP_PROXY = str(os.getenv("http_proxy", ""))
```

Comment:
Please:
1. uppercase it
2. add `aiogoogle_` prefix. Unless this is something GCE related?